app/client/src/pages/Editor/QueryEditor/EditorJSONtoForm.tsx

```
@@ -241,6 +245,35 @@ export function EditorJSONtoForm(props: Props) {
+    isFeatureEnabled,
+    currentActionConfig?.userPermissions,
+  );
+
+  // get the current action's plugin name
+  const currentActionPluginName = useSelector((state: AppState) =>
+    getPluginNameFromId(state, currentActionConfig?.pluginId || ""),
+  );
+
+  let actionBody = "";
+
+  if (!!currentActionConfig?.actionConfiguration) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)